engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorContextImpl.java

```diff
@@ -107,6 +111,12 @@ public HibernateValidatorContext clockProvider(ClockProvider clockProvider) {
 	}
 
 	@Override
+	public HibernateValidatorContext addValueExtractor(ValueExtractor<?> extractor) {
+		valueExtractors.add( extractor );
```

Comment:
I'll take a look. It may not work for `Configuration` though as the public method mandates `VE` there.